wowup-electron/src/app/pages/my-addons/my-addons.component.ts

```diff
@@ -395,6 +395,15 @@ export class MyAddonsComponent implements OnInit, OnDestroy, AfterViewInit {
     }
   }
 
+    public onVisitProvider(addon: Addon) {
```

Comment:
This can likely be replaced by the appExternalLink directive

Reply:
method is removed from ts - no longer needed